test/features/proof_of_address.feature

```diff
@@ -0,0 +1,151 @@
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    When I click on benefits_letter_cell (POA)
+    When I click on continue_button (POA)
+    When I upload national_identity_card
+    Then I should see poa_upload_confirmation_title (POA)
+    Then I should see poa_upload_confirmation_subtitle (POA)
+
+  Scenario: Test presence of the UI elements of the second screen of the PoA flow
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    Then I should see verify_uk_address_title (POA)
+    Then I should see need_a_doc_text (POA)
+    Then I should see poa_requirements (POA)
+
+  Scenario: Test navigation to the document selection screen
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see document_selection_screen_title (POA)
+    Then I should see document_selection_screen_subtitle (POA)
+
+  Scenario: Test presence of the UI elements of the bank/building society statement cell
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see bank_building_statement_icon (POA)
+    Then I should see bank_building_statement_title (POA)
+    Then I should see bank_building_statement_subtitle (POA)
+
+  Scenario: Test presence of the UI elements of the utility bill cell
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see utility_bill_icon (POA)
+    Then I should see utility_bill_title (POA)
+    Then I should see utility_bill_subtitle (POA)
+
+  Scenario: Test presence of the UI elements of the council tax letter cell
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see council_tax_letter_icon (POA)
+    Then I should see council_tax_letter_title (POA)
+
+  Scenario: Test presence of the UI elements of the benefits letter cell
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see benefits_letter_icon (POA)
+    Then I should see benefits_letter_title (POA)
+    Then I should see benefits_letter_subtitle (POA)
+
+  Scenario: Test navigation to the bank statement upload intro screen
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    When I click on bank_building_statement_cell (POA)
+    Then I should see document_upload_intro_screen_bank_statement_title (POA)
```

Comment:
I think testing the intro for one document type is good enough. Otherwise it will take ages to complete the test suite if we check for every single UI element

Reply:
Also this could refactored to use Scenario Outline with the different document types to avoid unnecessary duplication